frontend/src/Context/PublisherProvider/usePublisherOptions/usePublisherOptions.tsx

```diff
@@ -7,49 +7,57 @@ import {
 } from '@vonage/client-sdk-video';
 import useUserContext from '../../../hooks/useUserContext';
 import getInitials from '../../../utils/getInitials';
+import DeviceManager from '../../../utils/DeviceManager';
 
 /**
  * React hook to get PublisherProperties combining default options and options set in UserContext
- * @returns {PublisherProperties} publisher properties object
+ * @returns {PublisherProperties | null} publisher properties object
```

Comment:
What's the benefit of making this nullable, as opposed to having the caller use an empty object?